Doc/library/cmd.rst

```diff
@@ -35,6 +42,9 @@ interface.
    :attr:`use_rawinput` attribute to ``False``, otherwise *stdin* will be
    ignored.
 
+   .. versionchanged:: 3.11
```

Comment:
I want to make sure of this... 
the `readline.backend` (https://github.com/python/cpython/pull/112511) will be added `.. versionadded:: 3.13` but this will be changed 3.11.
So in this case, 3.11-3.12 backport will have a different code.. Am I right?

Reply:
Oh, sorry! I missed the number.
It definitely won't be changed in `3.11.0` or `3.12.0`, so this does need to be `3.13`.
I'm not convinced that this should be backported to earlier versions (3.11.7 and 3.12.2), but, that can be discussed on the issue.
The backports would need to adapt to the missing API.
```suggestion
   .. versionchanged:: 3.13
```